Bluetooth/nRF51822/Arduino/BLE-Buzzy-Pixel/BLE-Buzzy-Pixel.ino

```diff
@@ -53,7 +33,7 @@ int selfLED = 4; //TODO: Move to config.h
 
```

Comment:
selfLED should be moved to config.h or can be removed or its reference changed to DEVICE_ID